### ISSUE TITLE
how to set ssl client_key / client_cert ?

### ISSUE THREAD

**Issue by grosser** [toland/patron#173]
https://github.com/toland/patron/issues/84 is kinda similar, more docs in the readme would be nice

**Comment by julik** [comment]
@grosser Thanks for the issue, apologies I am finding it so late, I wasn't monitoring the patron repo as eagerly as I should have. I'll have a look, seems like we... don't have that option yet.

**Comment by julik** [comment]
Yes the PR mentioned does contain the options. Maybe it could be handy to rename the accessor methods for it to indicate that it is _not_ the self-signed _server_ certificate/CA but instead the _client_ SSL cert. @grosser I can offer to finish the PR if you really need the functionality or you can pick it up if you have some spare time, it looks practically done

**Comment by grosser** [comment]
I'm not blocked for now, I was researching alternatives for our current net-http-persistent setup  and this looked promising, I just could not get ssl to work :)

**Comment by julik** [comment]
Gotcha. For your use case: you need client certificates, a custom CA store, or both?

**Comment by grosser** [comment]
client cert+key  see https://gist.github.com/grosser/db29a5d4bb268d097b6154ef85ff7478
